drv/user-leds/src/main.rs

```diff
@@ -196,10 +191,15 @@ fn main() -> ! {
         for &led in config.blink_at_start {
             blinking[led] = true;
         }
+        if !config.blink_at_start.is_empty() {
```

Comment:
```suggestion
        if config.blink_at_start.is_some() {
```
?

Reply:
It's a slice, I don't think there's an `is_some` equivalent.

Reply:
aha, objection retracted